src/components/cc-select/cc-select.js

```diff
@@ -232,6 +237,14 @@ export class CcSelect extends LitElement {
           box-shadow: 0 0 0 .2em rgba(50, 115, 220, .25);
           outline: 0;
         }
+        
+        select.error {
+          border-color: var(--cc-color-border-danger) !important;
```

Comment:
the `!important` is not necessary considering the selector specificity but since it's the same style / code as `cc-input-*` I chose to keep it.
WDYT ?

Reply:
Agree, although we should try to use it as little as possible imo.